lib/krikri/harvester.rb

```diff
@@ -77,5 +84,40 @@ def run
       true
     end
 
+    ##
+    # An application-wide registry of defined Harvesters.
+    class Registry
```

Comment:
I could move this to `Krikri::Registry`, and have the harvester and mapper registries extend it.
There's not much difference, other than that I added a `#keys` method for the harvester one.